### PR TITLE
Maura's diagnostic

### PR DESCRIPTION
**Pull request by ladypinetree**
Comfort 2
Clarity 3

Starting to get it but I'm still VERY slow so I ran out of time.

### CODE REVIEW

File: diagnostic.md [hunk 1]
```diff
@@ -36,8 +36,7 @@ Now, we want to get a **single donut** from the server. Write the `curl` request
 you'd use to retrieve a **single donut**, using whatever ID you'd like.
 
 ```sh
-# your answer here
-```
+curl --include --request GET "http://www.example.com/books/7"
```
Comment (BenGitsCode) [inline diagnostic.md:40]: books? I thought we wanted donuts.

File: diagnostic.md [hunk 2]
```diff
@@ -21,8 +21,8 @@ Write the `curl` request you'd use to retrieve a list of **all donuts** on the
 server.
 
 ```sh
-# your answer here
-```
+
+curl http://www.example.com
 
 Write an AJAX request to retrieve a list of **all donuts** on the server.
```
Comment (BenGitsCode) [inline diagnostic.md:28]:
```js
let getDonuts = function () {
  return $.ajax({
    url: 'http://www.example.com/donuts',
    method: 'GET',
  });
};
```

**Review by BenGitsCode — COMMENTED** [review]
Please review solution branch [here](https://github.com/ga-wdi-boston/jquery-ajax-diagnostic/blob/016/solution/diagnostic.md)

File: diagnostic.md [hunk 3]
```diff
@@ -51,8 +50,7 @@ Write the `curl` request you'd use to delete a single donut, using whatever
 ID you'd like.
 
 ```sh
-# your answer here
-```
+curl --include --request DELETE "http://www.example.com/5"
```
Comment (bernardlee) [inline diagnostic.md:53]: The resource `donuts` is missing.

`curl --include --request DELETE http://www.example.com/donuts/1`

File: diagnostic.md [hunk 4]
```diff
@@ -51,8 +50,7 @@ Write the `curl` request you'd use to delete a single donut, using whatever
...
 
 Write an AJAX request to delete a single donut from the server.
```
Comment (bernardlee) [inline diagnostic.md:55]:
```javascript
let deleteDonut = function (donutID) {
  return $.ajax({
    url: 'http://www.example.com/donuts/' + donutID,
    method: 'DELETE',
  });
};
```

File: diagnostic.md [hunk 5]
```diff
@@ -72,9 +70,14 @@ data in JSON format.
 }
 ```
 
-```sh
-# your answer here
-```
+curl --include --request PATCH http://www.example.com/$ID \
```
Comment (bernardlee) [inline diagnostic.md:73]: The verb should be `POST` and the resource `donuts` is missing from the URL.

File: diagnostic.md [hunk 6]
```diff
@@ -72,9 +70,14 @@ data in JSON format.
...
+--header "Content-Type: application/json" \
+--data '{
+  "donut": {
+    "name": "French Cruller",
+    "price": "$0.99"
+  }
+}
```
Comment (bernardlee) [inline diagnostic.md:80]: Missing closing single quotation mark.

File: diagnostic.md [hunk 7]
```diff
@@ -100,8 +103,13 @@ object.
 ```
 
 ```sh
-# your answer here
-```
+curl --include --request PATCH http://www.example.com/42 \
```
Comment (bernardlee) [inline diagnostic.md:106]: Missing `donuts` resource.

File: diagnostic.md [hunk 8]
```diff
@@ -100,8 +103,13 @@ object.
...
+--header "Content-Type: application/json" \
+--data '{
+  "donut": {
+    "name": "Krüller"
+  }
+}'
 
 Write an AJAX request to change the donut on the server using JSON.
```
Comment (bernardlee) [inline diagnostic.md:115]:
```javascript
let changeDonut = function (donutID) {
  return $.ajax({
    url: 'http://example.com/donuts/' + donutID,
    method: 'PATCH',
    data: {
      "donut" {
        "name": "Krüller",
      }
    }
  });
};
```

**Comment by bernardlee** [comment]
0/10

**Pull request closed without merging**